Domains/Smartmetering/smartmeteringwebservices/Bundle.md

```diff
@@ -3,6 +3,8 @@
 ### Description
 Bundle is a special request in which one or more single request(s) to a specific device can be bundled. All request sent to this device make use of one communication channel, which may improve performance considerably.
 
+[GetBundleResponse](GetBundleResponse.md) returns the result of the actions of the bundle. The response request contains the DeviceIdentification and CorrelationUid which is received from the Bundle request.
+
 The Bundle request has an **Actions** tag. This contains a list of one or more single request(s).
 The response behavior is described in [ResponseMessages](./ResponseMessages.md).
```

Comment:
In the descriptions of the bundle actions, some things may need some attention.
* `GetSpecialDaysRequest` references `SetSpecialDays`. I guess we had better change the code since what is called `GetSpecialDaysRequest` in `sm-bundle.xsd` is actually a `SetSpecialDaysRequest`.
* `SetAdministrativeStatusRequest ` references `GetAdministrativeStatus`, but I think this should be `SetAdministrativeStatus`.
* `SetPushSetupSms` references `GetSetPushSetupSmsResponse`, which I believe should be `SetPushSetupSms`.